pkg/operator/ceph/pool/controller.go

```diff
@@ -137,13 +140,52 @@ func poolChanged(old, new cephv1.PoolSpec) bool {
 	return false
 }
 
+func (c *PoolController) handlePoolDelete(name, namespace string, retryInterval time.Duration) error {
+	retryCount := 0
+	var err error
+	var stats = new(ceph.PoolStatistics)
+	for {
+		logger.Infof("checking any images/snapshosts present in %v pool", name)
+		// check any PV is created in this cluster
+		stats, err = ceph.GetPoolStatistics(c.context, name, namespace)
+
+		if err != nil {
+			err = fmt.Errorf("failed to list images/snapshosts in pool %v with error %+v", name, err)
+			logger.Error(err)
+			return err
```

Comment:
```suggestion
			return fmt.Errorf("failed to list images/snapshosts in pool %v with error %+v", name, err)
```

And remove all the above?

Reply:
@leseb added above code to do logging